README.md

```diff
@@ -41,37 +41,143 @@ Indaba puts your stakeholder and expert network at your fingertips. It converts
 
 1. Set up a Docker Postgres container:
```

Comment:
Is there any installation for Docker that needs to happen before continuing with the steps?